state:

  # Upper and lower bounds for experimental space (meters, meters, radians). [x, y, theta]
  lower: [0.0, 0.0, 0] 
  upper: [20, 20, 6.28318531]

pred:
  # Total number of cars that you will be simulating. 
  total_number_of_cars: 1

  # Start position of cars(s) and list of potential goal locations

  # -- "Accurate" Example (going straight at a T-intersection) -- #
  car1_real_start_accurate: [11.5, 1, 1.57079633] 
  car1_real_goals_accurate: [[11.5, 19, 1.57079633], [1.0, 11.5, 3.1415927]] 
  # List of obstacles defined by [lower[x][y], upper[x][y]]
  car1_real_obstacles_accurate: [ [[13.0,0.0],[20.0,20.0]], [[0.0,13.0], [7.0,20.0]], [[0.0,0.0], [10.0,10.0]] ] 

  # -- "Unmodeled goal" Example (going on a dirt road off the side of the highway) -- #
  car1_real_start_goal: [11.5, 1, 1.57079633] 
  car1_real_goals_goal: [[11.5, 19, 1.57079633]] # true goal: [1.0, 11.5, 3.1415927]
  car1_real_obstacles_goal: [ [[13.0,0.0],[20.0,7.0]], [[13.0,13.0], [20.0,20.0]], [[0.0,0.0], [7.0,20.0]] ] 

  # -- "Unmodeled obstacle" Example (going around a pothole on the road) -- #
  car1_real_start_obstacle: [11.5, 1, 1.57079633] 
  car1_real_goals_obstacle: [[11.5, 19, 1.57079633]]
  car1_real_obstacles_obstacle: [ [[13.0,0.0],[20.0,20.0]], [[0.0,0.0], [7.0,20.0]] ] 

  # color coding for car visualization (RGB)
  car1_color: [1.0, 0.0, 0.0]

  # Dimensions of the prediction map
  sim_height: 20  # (num x states)
  sim_width: 20   # (num y states)
  sim_theta: 12   # (num theta states)

  # Num timesteps into the future to predict
  fwd_tsteps: 10

  # Initial betas depending on model
  beta_adaptive: [0.1, 0.3, 1.0, 3.0, 10.0]
  beta_rational: [0.05]
  beta_irrational: [10.0]

  # Car's height (in meters)
  car_height: 1.5

  # Car's driving speed (m/s) 
  car_vel: 1

  # Probability threshold 
  prob_thresh: 0.01

  # Process noise (epsilon-stubborn model)
  epsilon_dest: 0.02
  epsilon_beta: 0.02